cfgov/jinja2/v1/_includes/molecules/breadcrumbs.html

```diff
@@ -16,7 +16,7 @@
 {% macro render(breadcrumbs) %}
     <nav class="breadcrumbs" aria-label="Breadcrumbs">
         {% for crumb in breadcrumbs %}
-        <a class="breadcrumbs_link" href="{{ get_protected_url(crumb) }}">
+        <a class="breadcrumbs_link" href="{{ crumb.href if crumb.href else get_protected_url(crumb) }}">
```

Comment:
In most cases the `crumb` will be a page, but I'm using a workaround for Ask because its page hierarchy doesn't fit with the default approach to assembling breadcrumb items.